crates/sui-types/src/move_package.rs

```diff
@@ -185,6 +185,18 @@ pub struct UpgradeReceipt {
     pub package: ID,
 }
 
+#[derive(Eq, PartialEq, Clone, Debug, Serialize, Deserialize, JsonSchema)]
```

Comment:
This type seems like it's quite specific to JSON-RPC, it should probably in in `sui-json-rpc-types` or similar.